spec/openapi.yaml

```diff
@@ -118,6 +118,7 @@ x-tagGroups:
       - Commands V2
       - Enterprise Policy
       - Geofence
+      - Report
```

Comment:
In this file add a short summary.
name: Report
    description: APIs manage device active reports
    externalDocs:
      description: Find out more about Reports
      url: 'https://docs.esper.io/home/console.html#reports'